### PR TITLE
Add example host-guest complex setup and simulation using SMIRNOFF

### PR DESCRIPTION
**Pull request by davidlmobley**
This adds an example which sets up:
- generation of guest from SMILES
- 2D structure depiction of the guest
- 3D conformer generation/charging of guest
- Docking into the host
- 3D structure depiction of the complex
- solvation of the complex
- parameterization w/ SMIRNOFF99Frosst
- minimization and super short MD with OpenMM
- visualization of simulation

This could be extended in the future to also generate Yank inputs for binding free energy calculation, for example. 

As written this is fairly concise, but I think it's kind of cool and could enable research projects fairly easily I think.

@nhenriksen or @slochower , interested in reviewing for me?

### CODE REVIEW

**Comment by slochower** [comment]
Yes, sounds great. I can review tomorrow afternoon or Friday morning.

**Comment by davidlmobley** [comment]
@jchodera - builds are failing because your encrypted OE license on here is expired. Do you want to fix, or should I do it with mine?

**Comment by jchodera** [comment]
I'll do that right now!

**Comment by jchodera** [comment]
See #57.

File: examples/host_guest_simulation/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+Example provided by David Mobley (UCI)
+
+## Source materials
+- Based in part on a host-guest [docking Jupyter notebook](https://github.com/MobleyLab/SAMPL6/blob/master/host_guest/GenerateInputs.ipynb) I constructed for SAMPL6
+
+
+## Manifest
+- smirnoff_host_guest.ipynb
+- hbonds.ffxml: Force field XML file for constraining hbonds (from https://github.com/MobleyLab/SMIRNOFF_paper_code/blob/master/scripts/hbonds.ffxml, courtesy Andrea Rizzi )
+- OA.mol2: mol2 file of Octa Acid with AM1-BCC charges, as distributed with SAMPL5 (where it was labeled OAH.mol2). Atom types converted from GAFF to SYBYL via Antechember.
```
Comment (jchodera) [inline examples/host_guest_simulation/README.md]: Can we comment on how the protonation state of this form was chosen? It looks like all the carboxylic acids are negatively charged---is that what we expect?

File: examples/host_guest_simulation/README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+## Authorship
+
+Example provided by David Mobley (UCI)
+
+## Source materials
+- Based in part on a host-guest [docking Jupyter notebook](https://github.com/MobleyLab/SAMPL6/blob/master/host_guest/GenerateInputs.ipynb) I constructed for SAMPL6
+
+
+## Manifest
+- smirnoff_host_guest.ipynb
```
Comment (jchodera) [inline examples/host_guest_simulation/README.md]: Can you protect filenames with backticks?

**Comment by jchodera** [comment]
We don't have a way of automatically testing these Jupyter notebooks to make sure they run, do we? 

If not, there is a [plugin for `py.test` that we can use](https://github.com/computationalmodelling/nbval).

Once we start adding documentation too (which we should probably do soon!) we can use [nbsphinx](https://nbsphinx.readthedocs.io) to render the these notebooks as sphinx help.

File: examples/host_guest_simulation/README.md (hunk 1 above)
Comment (davidlmobley) [reply examples/host_guest_simulation/README.md]: It's controversial. -6 (hydrogen bonding across the bottom) or -8. No one knows. I can explain this in the notebook if you want.

**Comment by davidlmobley** [comment]
Currently no testing for jupyter notebooks here, @jchodera . That's on the list of things to do, but finishing the paper is a priority first.

File: examples/host_guest_simulation/README.md (hunk 2 above)
Comment (davidlmobley) [reply examples/host_guest_simulation/README.md]: Oops.

File: examples/host_guest_simulation/README.md (hunk 1 above)
Comment (davidlmobley) [reply examples/host_guest_simulation/README.md]: (Also, I can't spell -- I need to fix "Antechember" to "Antechamber".)

**Comment by slochower** [comment]
@davidlmobley Just getting to this today... I really appreciate the detailed instructions you put at the top of the `jupyter` notebook for the requirements. Incidentally, for unrelated reasons, I had to reset some of my `conda` environments I was glad to see your notes about installing the dependencies.

While trying to install `oeommtools`, I ran into the following error:
```bash
$ conda install -c nividic -c omnia oeommtools
Fetching package metadata ...............
Solving package specifications: .

UnsatisfiableError: The following specifications were found to be in conflict:
  - oeommtools -> python >=3.5,<3.6.0a0 -> openssl 1.0.1*
  - oeommtools -> python >=3.5,<3.6.0a0 -> xz 5.0.5
  - python 3.6*
Use "conda info <package>" to see the dependencies for each package.
```
...and I'm curious if you've come across before. 

---

To be clear, I didn't follow your instructions exactly. Because of how I had things setup on my machine previously, I used a hybrid approach that possibly messed things up. I created `environment.yaml` with:
```bash
name: openforcefield
channels:
  - anaconda
  - conda-forge
dependencies:
  - anaconda::numpy
  - anaconda::networkx
  - anaconda::lxml
  - anaconda::openmoltools
  - anaconda::parmed
  - anaconda::matplotlib
  - anaconda::pandas
  - anaconda::ipykernel
  - pip:
    - git+https://github.com/open-forcefield-group/openforcefield@548f04d005a51c03b40f1b4fa0ec32b80ae10775
    - https://pypi.anaconda.org/openeye/label/beta/simple/openeye-oenotebook/0.8.1/OpenEye-oenotebook-0.8.1.tar.gz
```
I installed that into a new `conda` environment, which requires adding 
```
[global]
extra-index-url = https://pypi.anaconda.org/OpenEye/simple
```
to `pip.conf`. Then I got the above error after trying `conda install -c nividic -c omnia oeommtools`.

**Comment by jchodera** [comment]
It would be really great to get `oemmtools` onto `conda-forge` to simplify this whole process. We can just make it a dependency of `openforcefield` at that point.

**Comment by davidlmobley** [comment]
I've not run across that before, @slochower . @nividic , can you take a look at this? It's the `oeommtools` dependency causing problems here. 

@slochower - might be useful to have the final contents of your environment for troubleshooting purposes.

**Comment by slochower** [comment]
Here are the packages in my `openforcefield` environment:

```bash
$ conda list
# packages in environment at /home/dslochower/data/applications/anaconda3/envs/openforcefield:
#
alabaster                 0.7.10                   py36_0    anaconda
ambermini                 16.16.0                       7    http://conda.binstar.org/omnia
babel                     2.5.0                    py36_0    anaconda
bleach                    2.0.0                     <pip>
cycler                    0.10.0                   py36_0    anaconda
dbus                      1.10.20                       0    anaconda
decorator                 4.1.2                    py36_0    anaconda
docutils                  0.14                     py36_0    anaconda
entrypoints               0.2.3                     <pip>
expat                     2.1.0                         0    anaconda
fftw3f                    3.3.4                         2    http://conda.binstar.org/omnia
fontconfig                2.12.1                        3    anaconda
freetype                  2.5.5                         2    anaconda
glib                      2.50.2                        1    anaconda
gst-plugins-base          1.8.0                         0    anaconda
gstreamer                 1.8.0                         0    anaconda
hdf5                      1.8.17                        2    anaconda
html5lib                  0.999999999               <pip>
icu                       54.1                          0    anaconda
imagesize                 0.7.1                    py36_0    anaconda
ipykernel                 4.6.1                    py36_0  
ipykernel                 4.6.1                     <pip>
ipython                   6.1.0                     <pip>
ipython                   6.1.0                    py36_0  
ipython-genutils          0.2.0                     <pip>
ipython_genutils          0.2.0                    py36_0  
ipywidgets                7.0.0                     <pip>
jedi                      0.10.2                    <pip>
jedi                      0.10.2                   py36_2  
jinja2                    2.9.6                    py36_0    anaconda
jpeg                      9b                            0    anaconda
jsonschema                2.6.0                     <pip>
jupyter                   1.0.0                     <pip>
jupyter-client            5.1.0                     <pip>
jupyter-console           5.2.0                     <pip>
jupyter-core              4.3.0                     <pip>
jupyter_client            5.1.0                    py36_0  
jupyter_core              4.3.0                    py36_0  
libffi                    3.2.1                         1    anaconda
libgcc                    5.2.0                         0    anaconda
libgfortran               3.0.0                         1    anaconda
libiconv                  1.14                          0    anaconda
libpng                    1.6.30                        1    anaconda
libsodium                 1.0.10                        0  
libxcb                    1.12                          1    anaconda
libxml2                   2.9.4                         0    anaconda
libxslt                   1.1.29                        0    anaconda
lxml                      3.8.0                    py36_0    anaconda
markupsafe                1.0                      py36_0    anaconda
matplotlib                2.0.2               np113py36_0    anaconda
mdtraj                    1.8.0                    py36_2    conda-forge
mistune                   0.7.4                     <pip>
mkl                       2017.0.3                      0    anaconda
nbconvert                 5.3.0                     <pip>
nbformat                  4.4.0                     <pip>
networkx                  1.11                     py36_0    anaconda
nose                      1.3.7                     <pip>
notebook                  5.0.0                     <pip>
numexpr                   2.6.2               np113py36_0    anaconda
numpy                     1.13.1                   py36_0    anaconda
numpydoc                  0.7.0                    py36_0    anaconda
OpenEye-oenotebook        0.8.1                     <pip>
OpenEye-toolkits          2017.6.1                  <pip>
OpenEye-toolkits-python3-linux-x64 2017.6.1                  <pip>
openforcefield            0.0.3                     <pip>
openmm                    7.1.1                    py36_0    http://conda.binstar.org/omnia
openmoltools              0.8.1                    py36_0    http://conda.binstar.org/omnia
openssl                   1.0.2l                        0    anaconda
pandas                    0.20.3                   py36_0    anaconda
pandocfilters             1.4.2                     <pip>
parmed                    2.7.3                    py36_1    http://conda.binstar.org/omnia
path.py                   10.3.1                   py36_0  
pcre                      8.39                          1    anaconda
pexpect                   4.2.1                    py36_0  
pexpect                   4.2.1                     <pip>
pickleshare               0.7.4                     <pip>
pickleshare               0.7.4                    py36_0  
pip                       9.0.1                    py36_1    anaconda
prompt-toolkit            1.0.15                    <pip>
prompt_toolkit            1.0.15                   py36_0  
ptyprocess                0.5.2                     <pip>
ptyprocess                0.5.2                    py36_0  
pygments                  2.2.0                    py36_0    anaconda
pyparsing                 2.2.0                    py36_0    anaconda
pyqt                      5.6.0                    py36_2    anaconda
pytables                  3.4.2               np113py36_0    anaconda
python                    3.6.2                         0    anaconda
python-dateutil           2.6.1                    py36_0    anaconda
pytz                      2017.2                   py36_0    anaconda
pyzmq                     16.0.2                    <pip>
pyzmq                     16.0.2                   py36_0  
qt                        5.6.2                         5    anaconda
qtconsole                 4.3.1                     <pip>
readline                  6.2                           2    anaconda
requests                  2.14.2                   py36_0    anaconda
scipy                     0.19.1              np113py36_0    anaconda
scripttest                1.3                       <pip>
setuptools                27.2.0                   py36_0    anaconda
simplegeneric             0.8.1                    py36_1  
simplegeneric             0.8.1                     <pip>
sip                       4.18                     py36_0    anaconda
six                       1.10.0                   py36_0    anaconda
snowballstemmer           1.2.1                    py36_0    anaconda
sphinx                    1.6.3                    py36_0    anaconda
sphinxcontrib             1.0                      py36_0    anaconda
sphinxcontrib-websupport  1.0.1                    py36_0    anaconda
sqlite                    3.13.0                        0    anaconda
terminado                 0.6                       <pip>
testpath                  0.3.1                     <pip>
tk                        8.5.18                        0    anaconda
tornado                   4.5.2                    py36_0  
tornado                   4.5.2                     <pip>
traitlets                 4.3.2                    py36_0  
traitlets                 4.3.2                     <pip>
wcwidth                   0.1.7                    py36_0  
wcwidth                   0.1.7                     <pip>
webencodings              0.5.1                     <pip>
wheel                     0.29.0                   py36_0    anaconda
widgetsnbextension        3.0.2                     <pip>
xz                        5.2.3                         0    anaconda
zeromq                    4.1.5                         0  
zlib                      1.2.11                        0    anaconda
```

**Comment by davidlmobley** [comment]
@slochower - do you want to try again in a clean conda `env`? Gaetano ( @nividic ) and I do not have these issues with `oeommtools`, and it also passes Travis tests (and the https://github.com/oess/openmm_orion repo, which depends on it, also passes tests). So we suspect the problem is on your end but neither of us has had a chance to dig into your details.

**Comment by slochower** [comment]
@davidlmobley Sure -- I'll try to start from a new environment this afternoon and get back to you.

**Comment by nhenriksen** [comment]
fwiw, I tried with a fresh conda environment on friday and got the same problem.

**Comment by davidlmobley** [comment]
Well, that's very odd. I don't get how it could be passing on Travis on https://github.com/oess/openmm_orion then. ( @nividic ? ) 

I'll have to look into this but it may take me a while to get to it.

**Comment by nividic** [comment]
@davidlmobley  and @slochower. I've just finished to test the oeommtools installation on Linux 14.04 in a virtual machine. I started from scratch installing conda and then I installed the oeommtools package as specified on the git hub repo. I was not able to reproduce the problem in any way. Currently, I'm using the package also on OS X and Linux 16. @slochower please if you have time, can you try to install the package as I did in a virtual machine or in a different computer? If this is taking to much time try to remove completely conda and reinstall it from scratch, then try to install just the oeommtools package.

**Comment by slochower** [comment]
@nividic thanks. I'm trying with Ubuntu 16.04.3 LTS. I'm going to give installation another shot this afternoon. One thought -- perhaps we need to install a `oemmtools` before the other dependencies?

**Comment by jchodera** [comment]
Is this a python 3.6 specific issue? The [`oess/oemmtools` travis](https://github.com/oess/oeommtools/blob/master/.travis.yml#L6) only tests python 3.5, but @slochower's issue is a 3.6 issue:
```
$ conda install -c nividic -c omnia oeommtools
Fetching package metadata ...............
Solving package specifications: .

UnsatisfiableError: The following specifications were found to be in conflict:
  - oeommtools -> python >=3.5,<3.6.0a0 -> openssl 1.0.1*
  - oeommtools -> python >=3.5,<3.6.0a0 -> xz 5.0.5
  - python 3.6*
Use "conda info <package>" to see the dependencies for each package.
```
I think that either means (1) there is something non-3.6-compliant, or, more likely, (2) a different version of a dependency is being picked up when 3.6 is used than when 3.5 is used on travis.

**Comment by nividic** [comment]
I retested the openforcefield installation with oeommtools in a python 3.5 environment and I do not have any problems. @jchodera I'm going to test python 3.6

**Comment by nividic** [comment]
The problem is python 3.6...I'm working on it

**Comment by nividic** [comment]
@davidlmobley @slochower @nhenriksen The problem with python 3.6 should have been fix now

**Comment by slochower** [comment]
Should I still use the same command (`$ conda install -c nividic -c omnia oeommtools`) to install? Or do I need to grab directly from your Github? I tried a second ago and it failed in a new test environment.

```bash
$ conda list
# packages in environment at /home/dslochower/data/applications/anaconda3/envs/testenvironment:
#
openssl                   1.0.2l                        0  
pip                       9.0.1                    py36_1  
python                    3.6.2                         0  
readline                  6.2                           2  
setuptools                27.2.0                   py36_0  
sqlite                    3.13.0                        0  
tk                        8.5.18                        0  
wheel                     0.29.0                   py36_0  
xz                        5.2.3                         0  
zlib                      1.2.11                        0  
(testenvironment) Tue Sep 05 08:27:48 warthog:~/data/projects/openforcefield/openforcefield-binding/build
$ conda install -c nividic -c omnia oeommtools
Fetching package metadata ...............
Solving package specifications: .

UnsatisfiableError: The following specifications were found to be in conflict:
  - oeommtools -> python >=3.5,<3.6.0a0 -> openssl 1.0.1*
  - oeommtools -> python >=3.5,<3.6.0a0 -> xz 5.0.5
  - python 3.6*
Use "conda info <package>" to see the dependencies for each package.
```

**Comment by davidlmobley** [comment]
@slochower - I think they've migrated to this one: `conda install -c OpenEye/label/Orion -c omnia oeommtools`

(I need to update my PR to reflect this.)

**Comment by slochower** [comment]
@davidlmobley @nividic Great! That worked.

**Comment by davidlmobley** [comment]
Yay! And you guys are up and running with the OpenEye tools again?

**Comment by jchodera** [comment]
@nividic : It would be great if you could cut separate releases (even just minor `x.y.z` updates of the `z`) and match the version numbers for each `conda` package you upload. It gets to be pretty hard to update the packages when bugfixes are released otherwise.
https://anaconda.org/OpenEye/oeommtools/files

**Comment by slochower** [comment]
Yup, should be. This'll be first test.

**Comment by davidlmobley** [comment]
(LMK if you have issues when testing it out.)

**Comment by slochower** [comment]
@davidlmobley  This may be a separate issue, but...

I created a brand new environment, installed `oemmtools` as above (`conda install -c OpenEye/label/Orion -c omnia oeommtools`). Then installed openforcefield using the suggested command (`conda install -c omnia openforcefield`) and finally tried to grab openeye-oenotebook using `pip install -i https://pypi.anaconda.org/openeye/simple openeye-oenotebook`. I got:

```
$ pip install -i https://pypi.anaconda.org/openeye/simple openeye-oenotebook
Collecting openeye-oenotebook
  Could not find a version that satisfies the requirement openeye-oenotebook (from versions: )
No matching distribution found for openeye-oenotebook
```
Is this another python 3.5 vs. 3.6 issue?

**Comment by nhenriksen** [comment]
can confirm.

Also related to the instructions in the PR @davidlmobley, I had trouble with `pip install -i https://pypi.anaconda.org/openeye/simple openeye-oenotebook` ("Could not find a version that satisfies ...").  I solved this by finding a .whl file and downloading and installing that:  `pip -v install OpenEye_oenotebook-0.8.1-py2.py3-none-any.whl`

**Comment by nividic** [comment]
@jchodera I'm going to set the oeommtools git hub tag as the conda repo version and I'll keep them in synchro.

**Comment by davidlmobley** [comment]
@mattgeballe - some issues with oenotebook noted here: https://github.com/open-forcefield-group/openforcefield/pull/56#issuecomment-327309920 -- maybe there is no Python 3.6 version yet? Can you comment?

**Comment by nividic** [comment]
In order to install the latest packmol release, I have temporary added my conda channel to the installation string:
```
conda install -c openeye/label/Orion oeommtools -c nividic -c omnia
```
When packmol will be available on the omnia channel  I'll remove my repo...I'm sorry about that

**Comment by davidlmobley** [comment]
@nhenriksen @slochower - so this is mostly working for you, except that I need to add (what info exactly?) to the Jupyter notebook? I'm not familiar with `whl` files so maybe you can tell me what I need to tell people...?

**Comment by jchodera** [comment]
> When packmol will be available on the omnia channel I'll remove my repo...I'm sorry about that

Sorry for the delay---looks like @andrrizzi and @lnaden were tied up. I can do this now.

Do you want 17.221 from https://github.com/mcubeg/packmol/releases?

**Comment by jchodera** [comment]
Working on getting the new `packmol` built [here](https://github.com/omnia-md/conda-recipes/pull/812).

**Comment by nhenriksen** [comment]
@davidlmobley I think the problem stems from the fact that the openeye-oenotebook package is not listed on https://pypi.anaconda.org/openeye/simple/ .  So it can't be found by running `pip install -i https://pypi.anaconda.org/openeye/simple openeye-oenotebook`.

The approach I used is fixed to a specific release version that I found on here: https://anaconda.org/OpenEye/OpenEye-oenotebook/files

To install, run:
`pip install https://anaconda.org/OpenEye/OpenEye-oenotebook/0.8.1/download/OpenEye_oenotebook-0.8.1-py2.py3-none-any.whl`

**Comment by nividic** [comment]
@jchodera YES please the 17.221 would be ok

**Comment by davidlmobley** [comment]
@nhenriksen - 
> @davidlmobley I think the problem stems from the fact that the openeye-oenotebook package is not listed on https://pypi.anaconda.org/openeye/simple/ . So it can't be found by running pip install -i https://pypi.anaconda.org/openeye/simple openeye-oenotebook.

Now that you mention it, I remember noticing the same thing. I think I installed via a download link from the website, which was a bit annoying; I never figured out how to do it via pip. @mattgeballe -- any prospects of making the "obvious" pip method for installing (`pip install -i https://pypi.anaconda.org/openeye/simple openeye-oenotebook`) work properly?

**Comment by slochower** [comment]
I haven't tested it yet, but I wonder if linking to `https://pypi.anaconda.org/openeye/label/beta/simple/openeye-oenotebook` instead of `https://pypi.anaconda.org/openeye/simple` will get the job done because I do see relevant-looking packages at the former URL.

**Comment by nividic** [comment]
@jchodera Thanks for packmol. I have noted that the packmol version that you selected is "1!17.221". Is it what you wanted or the "!" is a typo?

**Comment by Lnaden** [comment]
@nividic thats actually for setting the [Version Epoch](http://legacy.python.org/dev/peps/pep-0440/#version-epochs)

**Comment by jchodera** [comment]
This should ensure that this version is grabbed by default instead of the older date-based version. Let me know if that doesn't work.

**Comment by slochower** [comment]
@davidlmobley Thanks for working on this! I forgot to pick things up while waiting on install instructions for `openeye-oenotebook`, but I can do the manual install -- https://github.com/open-forcefield-group/openforcefield/pull/56#issuecomment-327574956 -- and test the notebook, if you think it'd still be useful.

**Comment by davidlmobley** [comment]
@slochower - I think it's working now, or at least, it sounded like it was for @nhenriksen . Maybe you can just doublecheck with him? If not it would be good to have one of you confirm, but if he's already checked I see no reason for you to doublecheck. 

Though, you may also want to try this out since this should give you an easy way to do on-the-fly modification of force field parameters for your host guest systems, which sounds like it would help you. (My recollection is it's as simple as something like `ff.changeParameter(...)` or similar).

The better long-term solution, of course, is for me to implement automated testing of the provided Jupyter notebooks under multiple Python versions, but it'll take a while to get to that.

**Comment by nhenriksen** [comment]
@davidlmobley just to clarify, I did get `openeye-oenotebook` installed correctly, but I wasn't able finish testing the whole notebook yet.

**Comment by davidlmobley** [comment]
OK, thanks. So @nhenriksen or @slochower - it would be great if one of you could test. It'd be good to make sure I didn't forget any prereqs or something. :)

**Comment by nhenriksen** [comment]
@davidlmobley I finally got around to giving this a go with CB7 as the host and heptane as a guest, and I have a few comments.

First, regarding prereqs. I still had some problem initially, but here is the final set of steps that got it working:
- Install Anaconda3
`./Anaconda3-4.4.0-Linux-x86_64.sh`
`conda update --all`
- Install OpenMM
- Install Prereqs
`conda install -c omnia openforcefield`
`conda install -c OpenEye/label/Orion -c omnia oeommtools`
`pip install -i https://pypi.anaconda.org/openeye/simple openeye-toolkits`
`pip install https://pypi.anaconda.org/openeye/label/beta/simple/openeye-oenotebook/0.8.1/OpenEye_oenotebook-0.8.1-py2.py3-none-any.whl`
`conda install -c OpenEye/label/Orion -c omnia pdbfixer`

Note that the `openeye-toolkits` step is not in your instructions and your wording for the `oeommtools` installation requires the user to add the package name to the end of command you give, which could be made simpler by just making the command complete.  Also, I used conda to install `pdbfixer`.

The rest of the notebook went fine, except that there seemed to be some problems with the CB7 topology.  Docking went fine, and the `complex.pdb` looks totally correct:
image

But the solvation step introduced some problems.  Somehow the coordinates of CB7 in `complex_solvated.pdb` got messed up, bizarrely starting in the middle of the molecule (i.e., about half the coordinates directly match `complex.pdb`, but then the rest appear to be shifted nonsensically).  The result is spaghetti CB7. I did check the CONECT lines and they appear to be correct.  Just the coordinates of some CB7 atoms have changed:
image

Despite the spaghetti, the minimization and MD completed, although it resulted in CB7 being twisted inside-out.  I was hoping it would be a mobius strip, but alas, it looks like just a regular twist:
image

So it looks like the CB7 topology got scrambled somehow, but other than that, things worked pretty well.  I've attached the CB7 `mol2` for debugging.
[cb7.tar.gz](https://github.com/open-forcefield-group/openforcefield/files/1357426/cb7.tar.gz)

**Comment by davidlmobley** [comment]
Thanks for testing, @nhenriksen . I've noted this and will revisit. Somewhat weird you had these problems and I didn't; presumably a version of something has changed/is different.

**Comment by davidlmobley** [comment]
@nhenriksen - I'm trying to look into the issue you raised here, but I can't extract the cb7 file you used. Can you re-attach if you still have it, or say where you got it from?

**Comment by nhenriksen** [comment]
@davidlmobley oops, I have no idea what I did to that file .... possibly tar'ed something twice.  See if this works.
[cb7.tar.gz](https://github.com/openforcefield/openforcefield/files/1489567/cb7.tar.gz)

**Comment by davidlmobley** [comment]
Thanks. In the meantime I was able to reproduce the issue (while offline) using files from benchmarksets. Working on sorting it out.

**Comment by davidlmobley** [comment]
@nhenriksen I believe I have fixed your issue; see #66 . Basically it's just that the solvation tools are sort of finicky and I'd botched something.

**Pull request merged**